### DISCUSSION TITLE
Programatically scroll Column or ListView

### DISCUSSION THREAD

**Discussion by ChristianKranemann** [flet-dev/flet#1026] [Q&A]
Hey flet community, 

i am trying to implement arrow up and down events on a column in order to move through the items. 
I got it working so far, but i am struggling how to move the contents of the list up or down if the items are out side of the visible space. 

I tried to use the Offset property, but this has some issues. For the column, it does not move the scroll bar. The scrollbar is fixed but the contend is moved. 

How can i programmatically move the scroll position?

Thanks in advance!

**Comment by ndonkoHenri** [comment]
There's an opened issue for that: https://github.com/flet-dev/flet/issues/566
